components/web_server.rst

```diff
@@ -10,20 +10,28 @@ The ``web_server`` component creates a simple web server on the node that can be
 through any browser and a simple :ref:`api-rest`. Please note that enabling this component
 will take up *a lot* of memory and may decrease stability, especially on ESP8266.
 
+**Web server version 1**
+------------------------
+
 .. figure:: /components/images/web_server.png
     :align: center
     :width: 86.0%
 
-    Web server version 1
-
+    
 
+**Web server version 2**
+------------------------
 .. figure:: /components/images/web_server-v2.png
     :align: center
     :width: 86.0%
 
-    Web server version 2
-
+**Web server version 3**
+------------------------
+.. figure:: /components/images/web_server-v3.png
+    :align: center
+    :width: 86.0%
 
+    
```

Comment:
I think I prefer these as the image labels rather than new headings on the page.
Also the v3 image seems to be broken?

Reply:
ok reverted that, but it looks like the caption of a figure is printed after the image which could  cause some confiusion.

the image for v3 see,s to be too lange, sinze only 300px x 300px is alowed, but the ones for v1 and 2 are also larger